tests/components/sensor/test_device_condition.py

```diff
@@ -51,7 +51,6 @@ def test_matches_device_classes(device_class: SensorDeviceClass) -> None:
         SensorDeviceClass.BATTERY: "CONF_IS_BATTERY_LEVEL",
         SensorDeviceClass.CO: "CONF_IS_CO",
         SensorDeviceClass.CO2: "CONF_IS_CO2",
-        SensorDeviceClass.CONDUCTIVITY: "CONF_IS_CONDUCTIVITY",
```

Comment:
These lists are only meant for exception to the standard naming.
ie. `carbon_monoxide` => CO` and `energy_storage => ENERGY`